src/components/login/index.js

```diff
@@ -41,24 +43,26 @@ const LoginComponent = ({
             getLoginCode(email, getPasswordlessCode);
         }
     }
-
+    
     return (
         <div className={`${styles.loginWrapper} step-wrapper`}>
             <>
                 <div className={`${styles.innerWrapper}`}>
                     <div className={styles.loginCode}>
+                        {summitData?.logo && <img src={`${summitData?.logo}`} />}
```

Comment:
do we need to a CSS class as a marker so we could be able to tweak the image filter per site ?

Reply:
@smarcet just added